rest-service/manager_rest/storage/resource_models.py

```diff
@@ -169,6 +169,7 @@ class Deployment(SQLResourceBase):
     policy_triggers = db.Column(db.PickleType)
     policy_types = db.Column(db.PickleType)
     outputs = db.Column(db.PickleType(comparator=lambda *a: False))
+    capabilities = db.Column(db.PickleType(comparator=lambda *a: False))
```

Comment:
can this at least be jsonstring

Reply:
So, originally we didn't use JSONs in postgres because they didn't work in sqlite (unittests), and because we wanted to remain DB neutral (e.g. switching to MySQL or whatever), as this was a Potsgres only feature. If we want to pronounce that we're only using postgres, which is quite reasonable, seeing as how all of our non-python code is hell bent on doing PG specific stuff, then I'm all in favor. But for now, for consistency's sake, I think we should stick with this PickleRick, until we decide to change it everywhere.